testing/scalar-functions.test

```diff
@@ -243,3 +243,26 @@ do_execsql_test date-with-invalid-timezone {
   SELECT date('2023-05-18 15:30:45+25:00');
 } {{}}
 
+do_execsql_test unicode-a {
+  SELECT unicode('a');
+} {97}
+
+do_execsql_test unicode-emoji {
+  SELECT unicode('😊');
+} {128522}
+
+do_execsql_test unicode-empty {
+  SELECT unicode('');
+} {NULL}
```

Comment:
This test looks incorrect:

```
sqlite> SELECT unicode('');

```

Reply:
<img width="700" alt="image" src="https://github.com/user-attachments/assets/84cd2e90-ca0b-45f1-9520-90c598afeba4">
in what way?

Reply:
nvn